src/twisted/python/runtime.py

```diff
@@ -37,7 +38,7 @@ class Platform:
     """
 
     type = knownPlatforms.get(os.name)
-    seconds = staticmethod(_timeFunctions.get(type, time.time))  # type: ignore[arg-type]  # noqa
```

Comment:
Can you take out the #noqa here?  I added a bunch of those because twistedchecker complained about line lengths.  Now that we removed twistedchecker, and use black for reformatting, I think we can get rid of this.